sklearn/metrics/_plot/confusion_matrix.py

```diff
@@ -137,7 +137,8 @@ def plot_confusion_matrix(estimator, X, y_true, labels=None,
     Parameters
     ----------
     estimator : estimator instance
-        Trained classifier.
+        Fitted classifier, or a fitted pipeline in which the last estimator
```

Comment:
Is this more grammatically correct?

```suggestion
        Fitted classifier or a fitted pipeline in which the last estimator
```

When writing, I usually prefer not having a comma when there are two items.

Reply:
Good point! I removed the comma in my latest commit.